utils/githooks/README.md

```diff
@@ -52,8 +56,10 @@ If the check is unable to parse the version output, it will fail. Try running
 3. `flake8`
 4. `isort`
 5. `yamllint`
-6. `gofmt`
+6. `gofmt` or `golang-go`
```

Comment:
Where is `golang-go` used in the githooks?

Reply:
That is the package name you need to use if you are using Ubuntu WSL to run the githooks.

Reply:
Oh, I see. The documentation is lacking here. This documentation refers to `gofmt` as the "package" but that is not technically correct. `gofmt` is the binary used, but that is provided by `golang-go` or `golang`, etc. depending on the system.

Anyway, I'm okay with this and will update the README to fix this section in a separate PR.